### PR TITLE
Project setup pull request

### PR DESCRIPTION
**Pull request by Profsain**
Added .gitignore file
Added linter.yml file for HTML and CSS
Updated README.md file
Added Header text in index.html
Added Style for Header text in main.css
Added test.md file to .gitignore.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (keroloussamy) [inline .gitignore]: - [ ] Very good by adding `test.md` file, you can also add `node_modules`, so if you have any packages don't get uploaded also.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @Profsain, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly be sure to fix your linters errors always, so here you have one linter error you can click on the `Details` button and see what is the errors and fit it.
image





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @Profsain, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Profsain** [comment]
Thanks

**Pull request merged**